starter-code/index.js

```diff
@@ -1,10 +1,45 @@
-var SortedList = function() {};
-
-SortedList.prototype.add     = function(item) {}
-SortedList.prototype.get     = function(pos) {}
-SortedList.prototype.max     = function() {}
-SortedList.prototype.min     = function() {}
-SortedList.prototype.average = function() {}
-SortedList.prototype.sum     = function() {}
+var SortedList = function() {
+  this.items = [],
+  this.length = this.items.length
+};
+
+SortedList.prototype.add     = function(item) {
+this.items.push(item)
+this.items.sort(function(a,b){
+  return a - b
+})
+this.length ++;
+}
+SortedList.prototype.get     = function(pos) {
+  if(pos < 1 || pos > this.length){
+    throw Error ("OutOfBounds");
+  }
+ return this.items[pos-1]
+}
+SortedList.prototype.max     = function() {
+  return Math.max.apply(Math, this.items);
```

Comment:
Clever using the apply method. Another way you could have written it, is using  "reduce":
```js
  var maxNumber =
    this.list.reduce(
      (maxSoFar, number) => {
        return Math.max(maxSoFar, number);
      });
  return maxNumber;
```